server/config/config.go

```diff
@@ -832,6 +836,8 @@ const (
 	defaultHotRegionsReservedDays      = 7
 	// It means we skip the preparing stage after the 48 hours no matter if the store has finished preparing stage.
 	defaultMaxStorePreparingTime = 48 * time.Hour
+	// When a slow store affected more than 30% of total stores, it will trigger evicting.
```

Comment:
```suggestion
	// When a store's slowness exceeds 30% of total stores, it will trigger evicting.
```

Reply:
Use affected would be more accurate because:
If a store become slow (the latency still faster than 30% stores), but the `become slow` event affected more than 30% (30%+ stores' QPS dropped)